src/components/RecipeCard/RecipeCard.jsx

```diff
@@ -1,17 +1,48 @@
 import * as React from "react"
+import { useState } from "react"
+import { Alert } from "react-bootstrap" 
+import axios from 'axios';
 import "./RecipeCard.css"
 
 import RecipeModal from "../RecipeModal/RecipeModal"
 
 export default function RecipeCard(props) {
+  const recipeInfoUrl = "http://localhost:3001/apirecipeinfo/"
+  const [recipeModelShow, setRecipeModalShow] = useState(false)
+  const [recipeInfo, setRecipeInfo] = useState({})
+  const [error, setError] = useState("")
+
+  // onclick function for each recipe card
+  const handleRecipeCardClick = async (showStatus, recipeId) => {
+    setError("")
```

Comment:
Why are you setError("") here? Shouldn't it already be "" since that's what you originally set it to?